components/coaching/AboutSC.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <section class="container mx-auto px-6 md:px-8 mt-[83px]">
+    <div class="flex flex-wrap xl:flex-nowrap items-center justify-center">
+      <img
+        class="sc-image mb-6"
+        :src="pageData.fields.image.fields.file.url"
+      >
+      <div class="SC-text">
+        {{ pageData.fields.paragraph }}
+      </div>
```

Comment:
Same here. Likely a RichText component and needs to have the same styling as the other body text on all pages so it is consistent